DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/KeywordCampaign.java

```diff
@@ -27,20 +27,31 @@
 
     // settings
     public String name;
-
-    // TODO: add the rest of the input settings
+    public String fromDate;
+    public String toDate;
+    public double dailyBudget;
+    public String location;
```

Comment:
What is an example location? Might be good to specify how the location should be given/stored. Ex is it like "US" or maybe like "United States" and only for countries?

Reply:
In the design doc, I wrote a description of exactly what is required of the location. Once we get your approval on the settings we have in the design doc (I added settings descriptions last Wednesday), I will update the java classes with the finalized settings and add descriptive comments.

Reply:
By this comment, I meant it would be nice to include the specification of the location string as well, since not everyone will have access to the design doc.